site/docs/relations/logical_relations.md

```diff
@@ -302,6 +311,8 @@ The output type is as follows for the various operations
 | Intersection (Multiset) | (R, R, R, R, R, R, R, N)
 | Union Distinct          | (R, N, N, N, N, N, N, N)
 | Union All               | (R, N, N, N, N, N, N, N)
+| Minus Distinct          | (R, R, R, R, N, N, N, N)
+| Intersection Distinct   | (R, R, R, R, R, N, N, N)
```

Comment:
minor: Could you reorganize this to include to `Minus Distinct` with the other Minuses, and `Intersection Distinct` with the other Intersections. It's easier to read that way IMO.